contribs/gmf/apps/desktop/index.html

```diff
@@ -204,12 +204,11 @@
         </div>
       </div>
       <div class="gmf-app-map-container" ng-class="{'gmf-app-infobar-active': mainCtrl.showInfobar}">
-        <gmf-search gmf-search-map="mainCtrl.map"
-          gmf-search-datasources="mainCtrl.searchDatasources"
-          gmf-search-coordinatesprojections="mainCtrl.searchCoordinatesProjections"
-          gmf-search-colorchooser="false"
+        <gmf-search gmf-search-map="::mainCtrl.map"
```

Comment:
no real need, better to use `<` in bindings of `gmf-search`

Reply:
Same as my comment in the print. That's a one-time, not a one way